src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -87,23 +125,23 @@ public void angleClimbHook(DoubleSolenoid.Value value) {
     }
 
     public void extendFixedArm() {
-        setMotor(ClimbConstants.TEST_SPEED_EXTEND, climbFixedMotor);
+        setMotor(TEST_SPEED_EXTEND, climbFixedMotor);
```

Comment:
It would be preferable to read the speed from `testSpeedExtend`, as this would allow the user to change the value via shuffleboard.